.pre-commit-config.yaml

```diff
@@ -46,3 +46,4 @@ repos:
           - file
           - python
         exclude: vendored
+        args: [--config, python/setup.cfg]
```

Comment:
I don't think we need the absolute path here as precommit should run within the arrow dir by default?

Reply:
Yes, I assume so